src/apps/metadata/tool_requests.clj

```diff
@@ -228,7 +228,7 @@
 (defn- format-tool-request-dates
   [{:keys [date_submitted date_updated] :as tool-request}]
   (assoc tool-request :date_submitted (.getTime date_submitted)
-                      :date_updated   (.getTime date_updated)))
+         :date_updated   (.getTime date_updated)))
```

Comment:
I like that it will align k-v pairs under the map of an `(assoc map` line, but in this case it looks like the first k-v pair needs to be on a newline now, for consistency?

Reply:
Yeah, it's indenting `assoc` just like any other function call. Arguments on subsequent lines are aligned with the first argument on the first line, which happens to be the map itself in this case. Having the first k-v pair on a separate line seems reasonable. The only exception would be an assoc in a threading macro, I think. In that case, the newline can be optional.